src/main/java/aspects/MetricsAspect.java

```diff
@@ -61,14 +73,37 @@ public Object logRequest(ProceedingJoinPoint joinPoint) throws Throwable {
                 "domain:" + domain,
                 "user:" + user,
                 "request:" + requestPath,
-                "duration:" + timer.getDurationBucket()
+                "duration:" + timer.getDurationBucket(),
+                "blocked_time:" + getBlockedTime(),
+                "restore_blocked_time:" + getRestoreBlockedTime(),
+                "install_blocked_time:" + getInstallBlockedTime(),
+                "submit_blocked_time:" + getSubmitBlockedTime()
         );
         if (timer.durationInMs() >= 60 * 1000) {
             sendTimingWarningToSentry(timer);
         }
         return result;
     }
 
+    private String getBlockedTime() {
+        long blockedDuration = restoreFactory.getDownloadRestoreTimer().durationInMs() +
+                installService.getInstallTimer().durationInMs() +
+                submitService.getSubmitTimer().durationInMs();
+        return "blocked_total:" + blockedDuration;
+    }
+
+    private String getRestoreBlockedTime() {
+        return "restore_blocked:" + restoreFactory.getDownloadRestoreTimer().durationInMs();
+    }
+
+    private String getInstallBlockedTime() {
+        return "install_blocked" + installService.getInstallTimer().durationInMs();
```

Comment:
same as above except that if you did intend to keep this, I think you want a `:` after the tag.